cmd/servicelog/post.go

```diff
@@ -129,15 +133,26 @@ func (o *PostCmdOptions) Run() error {
 		return err
 	}
 
-	o.parseUserParameters() // parse all the '-p' user flags
-	o.readFilterFile()      // parse the ocm filters in file provided via '-f' flag
-	o.readTemplate()        // parse the given JSON template provided via '-t' flag
+	o.parseUserParameters()   // parse all the '-p' user flags
+	err := o.parseOverrides() // parse all the '-o' flags
+	if err != nil {
+		log.Fatalf("Error parsing overrides: %s", err)
+	}
+
+	o.readFilterFile() // parse the ocm filters in file provided via '-f' flag
+	o.readTemplate()   // parse the given JSON template provided via '-t' flag
 
 	// For every '-p' flag, replace its related placeholder in the template & filterFiles
 	for k := range userParameterNames {
 		o.replaceFlags(userParameterNames[k], userParameterValues[k])
 	}
 
+	// Replace any overrides
+	err = o.applyOverrides()
+	if err != nil {
+		log.Fatalf("Could not apply overrides: %s", err)
```

Comment:
If we were using `fmt.Errorf()` here then this would be correct, however this is the point where we log the error to the console using `logrus.Fatalf()` which does not support `%w`

There is a mixture of returning and fatally logging errors in this `Run()` function, do you want me to instead return an error?

Reply:
ah, ok, I am not aware of it, if this is how it works, let's keep it